compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -89,6 +89,17 @@ static std::string getTargetArch(func::FuncOp entryPoint) {
   return "";
 }
 
+static std::string getTarget(func::FuncOp entryPoint) {
```

Comment:
Not great to use string comparisons this way. Can we instead make this `isCuda` or something similar?

Reply:
I'm not sure this looks much better. I think Mahesh's point was to not have `"cuda"` string in the middle of some logic. I would at least make it a constant expression. and I think isolating the logic to check the target was better.
So basically either do:
```
if(isCudaTarget(entryPoint)
 ...
```
or:
```
if(isTarget(entrypoint, g_cudaTarget))
  ...
```